### ISSUE TITLE
Remote debugging

### ISSUE THREAD

**Issue by badsyntax** [redhat-developer/vscode-extension-tester#102]
It would be awesome if we could do remote debugging of the ui tests from within vscode. I'd like to be able to debug my application code while the tests are running.
Can you explore the possibility of using remote debugging and launching the tests from within vscode (via launch configurations) to allow us to achieve this?

**Comment by jrichter1** [comment]
I've added a debug configuration to the sample project
 - https://github.com/redhat-developer/vscode-extension-tester/blob/master/sample-projects/helloworld-sample/.vscode/launch.json

 and a wiki page with some info  
- https://github.com/redhat-developer/vscode-extension-tester/wiki/Debugging-Tests

Check it out, see if it works for you :)

**Comment by badsyntax** [comment]
Hey, thanks for this! I think i'm missing something though.

These are the steps I take:

1. `git clone https://github.com/redhat-developer/vscode-extension-tester.git`
2. `cd sample-projects/helloworld-sample/`
3. `npm i`
4. Open cwd in vscode
5. Run `Debug UI Tests`, and get the following error in the terminal output:

```
richardwillis:~/.../badsyntax/vscode-extension-tester/sample-projects/helloworld-sample (master *=)
$  cd /Users/richardwillis/Projects/badsyntax/vscode-extension-tester/sample-projects/helloworld-sample ; /Users/richardwillis/.nvm/versions/node/v12.16.2/bin/node --inspect-brk=18963 node_modules/.bin/extest setup-and-run /Users/richardwillis/Projects/badsyntax/vscode-extension-tester/sample-projects/helloworld-sample/out/ui-test/*.js -o settings.json --mocha_config /Users/richardwillis/Projects/badsyntax/vscode-extension-tester/sample-projects/helloworld-sample/src/ui-test/.mocharc-debug.js 
Debugger listening on ws://127.0.0.1:18963/80bd3737-629d-4188-8744-d010f9abb87c
For help, see: https://nodejs.org/en/docs/inspector
Debugger attached.
VS Code exists in local cache, skipping download
ChromeDriver 78.0.3904.105 exists in local cache, skipping download
/bin/sh: vsce: command not found
Error: Command failed: vsce package
    at checkExecSyncError (child_process.js:630:11)
    at Object.execSync (child_process.js:666:15)
    at CodeUtil.packageExtension (/Users/richardwillis/Projects/badsyntax/vscode-extension-tester/sample-projects/helloworld-sample/node_modules/vscode-extension-tester/out/util/codeUtil.js:146:23)
    at ExTester.<anonymous> (/Users/richardwillis/Projects/badsyntax/vscode-extension-tester/sample-projects/helloworld-sample/node_modules/vscode-extension-tester/out/extester.js:103:27)
    at Generator.next (<anonymous>)
    at /Users/richardwillis/Projects/badsyntax/vscode-extension-tester/sample-projects/helloworld-sample/node_modules/vscode-extension-tester/out/extester.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/Users/richardwillis/Projects/badsyntax/vscode-extension-tester/sample-projects/helloworld-sample/node_modules/vscode-extension-tester/out/extester.js:4:12)
    at ExTester.installVsix (/Users/richardwillis/Projects/badsyntax/vscode-extension-tester/sample-projects/helloworld-sample/node_modules/vscode-extension-tester/out/extester.js:84:16)
    at ExTester.<anonymous> (/Users/richardwillis/Projects/badsyntax/vscode-extension-tester/sample-projects/helloworld-sample/node_modules/vscode-extension-tester/out/extester.js:129:24)
Waiting for the debugger to disconnect...
Killed: 9
```

**Comment by jrichter1** [comment]
Okay, looks like environment variables are not propagating to VS Code terminal, I'll fix that.
In the meantime, you can work around it with `npm i -g vsce`

**Comment by jrichter1** [comment]
Fixed in 2.6.2
